### PR TITLE
Arithmetic - Simple Interest

### PR DESCRIPTION
**Pull request by Abduliitr**
## Checklist
- Solution is tested on sphere engine
    - [x] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [x] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [x] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [x] Author
    - [x] Reviewer

### CODE REVIEW

**Review by rpapavan97 — APPROVED** [review]

File: solutions/arithmetic/simple-interest.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let principal = parseInt(readLine());
+let time = parseInt(readLine());
+let rate = parseInt(readLine());
+
+let simpleInterest = (principal * time * rate)/100;
+
+console.log(parseInt(simpleInterest));
```
Comment (sailesh-ramayanam) [inline solutions/arithmetic/simple-interest.js:20]: This doesn't need conversion to int. I don't think there is a test case to check that scenario. Gives 100% score even without `parseInt`. Let us skip the conversion to keep it simple (as this is just first few days of bootcamp).

Comment (Abduliitr) [reply solutions/arithmetic/simple-interest.js:20]: Yes that's true. 

But keeping parseInt is making it consistent with other solutions..

If we remove parseInt, they need to know that js is auto-type-casting it from string to integer, which is relatively more difficult for them as of now..

Comment (Abduliitr) [reply solutions/arithmetic/simple-interest.js]: Removed.

Comment (Abduliitr) [reply solutions/arithmetic/simple-interest.js]: Removed.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**